invisible_cities/cities/maurilia_test.py

```diff
@@ -0,0 +1,71 @@
+"""
+code: maurilia_test.py
+description: test suite for maurilia
+author: Josh Renner
+IC core team: Jacek Generowicz, JJGC,
+G. Martinez, J.A. Hernando, J.M Benlloch
+package: invisible cities. See release notes and licence
+"""
+from __future__ import print_function
+from __future__ import absolute_import
+
+import os
+from os import path
+from glob import glob
+import tables as tb
+import numpy as np
+
+from pytest import mark
+
+from   invisible_cities.core.configure import configure
+import invisible_cities.core.tbl_functions as tbl
+import invisible_cities.core.wfm_functions as wfm
+import invisible_cities.core.system_of_units as units
+from   invisible_cities.sierpe import fee as FEE
+import invisible_cities.sierpe.blr as blr
+from   invisible_cities.database import load_db
+from   invisible_cities.cities.maurilia import Maurilia
+from   invisible_cities.core.random_sampling \
+     import NoiseSampler as SiPMsNoiseSampler
+
+config_file_format = """
+
+# set_input_files
+PATH_IN {PATH_IN}
+FILE_IN {FILE_IN}
+
+# set output file compression
+COMPRESSION {COMPRESSION}
+"""
+
+config_file_spec = dict(PATH_IN  = '$ICDIR/database/test_data/',
+                        FILE_IN  = 'NEW_se_mc_reco_pmaps_1evt.h5',
+                        COMPRESSION = 'ZLIB4')
+
+config_file_contents = config_file_format.format(**config_file_spec)
+
+@mark.serial
```

Comment:
The `serial` mark should be used in tests that cannot be run concurrently with other tests. The only reason for this that we have identified so far in IC is those tests which form a chain where some test uses files produced by another one.

I do not see why this test should be marked as serial.